server/services/RollupService.ts

```diff
@@ -236,31 +361,34 @@ export default class RollupService {
         rollup: hit._source,
         metadata: null,
       }));
-      let ids = rollups
-        .map((rollup) => {
-          return rollup._id;
-        })
-        .join(",");
-
-      const explainResponse = await this.explainRollup(req, h, ids);
-
-      if (explainResponse.ok) {
-        rollups.map((item) => {
-          item.metadata = explainResponse.response[item._id];
-        });
-        return {
-          ok: true,
-          response: { rollups: rollups, totalRollups: totalRollups, metadata: explainResponse.response },
-        };
-      } else {
-        return { ok: false, error: explainResponse.error };
-      }
+      let ids = "";
+      rollups.map((rollup) => {
```

Comment:
This can be cleaned up, what was reason for switching from that previous to this?
The previous one looks like it can be cleaned up too: `const ids = rollups.map(rollup => rollup._id).join(",")`

Reply:
Thanks for catching this. I accidentally used the old code when resolving merge conflicts. Simplifed to `const ids = rollups.map((rollup) => rollup._id).join(",");`